src/client/_clientwindow.py

```diff
@@ -1325,16 +1325,17 @@ def handle_mod_info(self, message):
         self.modInfo.emit(message)
 
     def handle_game_info(self, message):
-        if message['uid'] == self.game_session.game_uid:
-            self.game_session.game_map = message['mapname']
-            self.game_session.game_mod = message['featured_mod']
-            self.game_session.game_name = message['title']
-            self.game_session.game_visibility = message['visibility']
-
         if 'games' in message:
             for game in message['games']:
                 self.gameInfo.emit(game)
         else:
+            # sometimes we get the game_info message before a game session was created
+            if self.game_session and message['uid'] == self.game_session.game_uid:
```

Comment:
Yes. Although I am today going to split the pyqt5/python3.5 stuff away from the model/view adaptation. The former is ready to merge modulo the webview stuff, so it makes sense to sort that out before doing the other overhaul.